config.rb.sample

```diff
@@ -46,3 +46,7 @@
 #$vb_gui = false
 #$vb_memory = 1024
 #$vb_cpus = 1
+
+# What can we use to sync data
```

Comment:
actually, ^ might not be accurate. Can you just document whatever the behavior would be for `true` vs `false`? Will rsync be used, or will shared folders just be ignored?